### PR TITLE
Add files of HACKERRANK C Problems Set 2-Conditionals and Loops

### PR DESCRIPTION
**Pull request by adityagi02**

### CODE REVIEW

**Review by AkashSingh3031 — CHANGES_REQUESTED** [review]
Change format of code
like
```
int main()
{
    int i;
    scanf("%d",&i);

    if(i==1)
    {
        printf("one");
    }
    else if (i==2)
    {
        printf("two");
    }
}
```

**_write readable code_**

File: 3]. Competitive Programming/HackerRank/C/Conditionals and Loops/_04)_Printing_Pattern_Using_Loops.c [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#include <stdio.h>
```
Comment (AkashSingh3031) [inline 3]. Competitive Programming/HackerRank/C/Conditionals and Loops/_04)_Printing_Pattern_Using_Loops.c:1]: This code format is good

**Comment by adityagi02** [comment]
done brother 👍

**Review by adityagi02 — COMMENTED** [review]
completed , can be merged now👍

File: 3]. Competitive Programming/HackerRank/C/Conditionals and Loops/_04)_Printing_Pattern_Using_Loops.c (hunk 1 above)
Comment (adityagi02) [reply 3]. Competitive Programming/HackerRank/C/Conditionals and Loops/_04)_Printing_Pattern_Using_Loops.c:1]: okay

**Review by AkashSingh3031 — COMMENTED** [review]
Accepted

**Pull request merged**